charts/perses/templates/_helpers.tpl

```diff
@@ -66,3 +66,9 @@ Create the name of the service account to use
 {{- define "perses.dns" -}}
 http://{{ include "perses.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.service.targetPort }}
 {{- end -}}
+
+{{- define "perses.validateDatabaseConfig" -}}
```

Comment:
I believe we can use NOTEX.txt to validate that, wdyt? https://github.com/perses/helm-charts/blob/main/charts/perses/templates/NOTES.txt

Reply:
Sure. Thank you for the pointer. Fixed in 583d0e3e8b210c0b8d9abb0222a387d997e1a68e